tests/performance/Makefile

```diff
@@ -31,20 +31,19 @@ build: ## builds distributed osparc locust docker image
 push:
 	docker push itisfoundation/locust:$(LOCUST_VERSION)
 
-.PHONY: up down run
-up: ## runs distributed osparc locust using target=locust_test_file.py, optional host can be set
-	@$(call check_defined, target, please define target file when calling $@ - e.g. ```make $@ target=MY_LOCUST_FILE.py```)
-	@export LOCUST_FILE=$(target); \
-	export TARGET_URL=$(if $(host),$(host),"http://$(get_my_ip):9081"); \
-	docker compose --file docker-compose.yml up --scale worker=4
-
+.PHONY: down
 down: ## stops and removes osparc locust containers
 	docker compose --file docker-compose.yml down
 
 .PHONY: test
 test: ## runs osparc locust. locust and test confiuration are specified in .env file next to target file
 	@$(call check_defined, target, please define target file when calling $@ - e.g. ```make $@ target=MY_LOCUST_FILE.py```)
 	@export LOCUST_FILE=$(target); \
-	export ENV_FILE=$$(dirname $$(realpath locust_files/$${LOCUST_FILE}))/.env; \
-	if [ ! -f $${ENV_FILE} ]; then cp $$(dirname $${ENV_FILE})/.env-devel $${ENV_FILE}; fi; \
+	export ENV_FILE=$$(realpath .env); \
```

Comment:
If this needs human intervention, how is this going to work for the CI?

Reply:
Well I will need to first generate the settings (`.env` file) in the CI.
I.e. I will run something like 
```bash
python locust_files/ping_platform_test.py --LOCUST_HOST=<host name> > .env
make test target=locust_files/ping_platform_test.py
```